app/models/ebean/Organization.java

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright 2016 Smartsheet.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package models.ebean;
+
+import com.avaje.ebean.Model;
+import com.avaje.ebean.annotation.CreatedTimestamp;
+import com.avaje.ebean.annotation.UpdatedTimestamp;
+
+import java.sql.Timestamp;
+import java.util.UUID;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.Version;
+
+/**
+ * Data model for organizations.
+ *
+ * @author Brandon Arp (brandon dot arp at smartsheet dot com)
+ */
+// CHECKSTYLE.OFF: MemberNameCheck
+@Entity
+@Table(name = "organizations", schema = "portal")
+public class Organization extends Model {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "uuid")
+    private UUID uuid;
+
+    @Version
+    @Column(name = "version")
+    private Long version;
+
+    @CreatedTimestamp
+    @Column(name = "created_at")
+    private Timestamp createdAt;
+
+    @UpdatedTimestamp
+    @Column(name = "updated_at")
+    private Timestamp updatedAt;
+
+    @Column(name = "name")
```

Comment:
That's a very good point. However, I am not convinced it should come from this table. In fact, I think that relates more to how we pass session, authN and authZ information to the MP from whatever system is in front of it. 

Then I can see there be several ways to obtain this information. One could be from another table (e.g. organizationdetails; if the data _is_ mastered here). Another could be from the session and a third from a remote service (or other database). 

So, in my opinion the retrieval mechanism for organization details should be separate from the foreign key support we're building. Second, we should plan how to consume that information on MP to show things like what organization you are in.

However, I think adding name in this PR is putting the cart before the horse.